BranchGuide.md

```diff
@@ -0,0 +1,43 @@
+# Mixed Reality Toolkit Branch guide
+
+The Mixed Reality Toolkit uses multiple working branches within the project to manage the various components of delivery, these can be broken down to:
+
+---
+## Release Branches
+
+Release branches are frozen code linked to a specific released asset, the code held within these branches should match 1-1 with their corresponding asset.
+This ensures whether you download the asset or the code, there are no differences in the delivery.
+
+* **[master](https://github.com/Microsoft/MixedRealityToolkit-Unity) - [HoloToolkit 2017.4.2.0](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/tag/2017.4.2.0)** - Current HoloToolkit release branch.
+
+* **[mrtk_release](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release) - [Mixed Reality Toolkit 2018.9.0](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/tag/2018.9.0-Beta) (vNext Beta)** - Current Mixed Reality Toolkit (vNext) beta release.
+
+---
+## Development Branches
+
+Development branches relate to the current active and tested development state of the product.  These should be considered "stable", however bugs may be introduced during the course of development.
+
+* **Dev_Working_Branch/[htk_development](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/htk_development)** - Legacy HoloToolkit development - Currently frozen and due to be deprecated.
```

Comment:
This branch is not frozen. We are taking key customer fixes and releasing as needed.